packages/lesswrong/components/editor/CKPostEditor.tsx

```diff
@@ -386,6 +387,8 @@ const CKPostEditor = ({
   const webSocketUrl = ckEditorWebsocketUrlOverrideSetting.get() || ckEditorWebsocketUrlSetting.get();
   const ckEditorCloudConfigured = !!webSocketUrl;
   const initData = typeof(data) === "string" ? data : ""
+
+  const [_editors, setEditors] = useContext(EditorsContext);
```

Comment:
nit: why `_` prefix?

Reply:
Because it's unused and it signals that. Happy to change it to whatever you like (e.g. just `_`)